### ISSUE TITLE
Cannot navigate caption button language menu with accessibility 

### ISSUE THREAD

**Issue by robertmausser** [adaptlearning/adapt-contrib-media#232]
Caption button is not accessible through tab/ arrow keys. When user hits enter on it then nothing happens.

Caption button should be accessible through tab/ arrow keys. When user hits enter on it then the button should expand.

If caption button will not be accessible then screen reader and keyboard user will not be able to access its functionality.

https://www.screencast.com/t/mLvG21vtC

Provide proper tab indexing and aria role to caption button.

**Comment by swashbuck** [comment]
We can review this again as part of #299

**Comment by swashbuck** [comment]
Needs reviewing as part of #299

**Comment by kirsty-hames** [comment]
> Needs reviewing as part of [#299](https://github.com/adaptlearning/adapt-contrib-media/pull/299)

I've retested this since the PR and captions are still not fully accessible. I can toggle captions on/off fine using keyboard controls but the caption selector list (menu for caption languages) doesn't display when focused. Whilst I can tab onto the list and use arrow keys to select a language, the list is visually hidden but occasionally flickers display when navigating through the options. See screen recording below.

https://github.com/user-attachments/assets/11e785c4-2f3b-4ae3-810c-2beb197356cd

This is due to the `.mejs__offscreen` class appended to `.mejs__captions-selector`. [Issue has been raised for mediaelement](https://github.com/mediaelement/mediaelement/issues/2949). 

The caption selector list should remain on screen whilst it has focus. We could probably patch fix this with a CSS override until it's addressed in mediaelement.

**Comment by swashbuck** [comment]
> We could probably patch fix this with a CSS override until it's addressed in mediaelement.

@kirsty-hames I think that's a good solution considering the infrequency of mediaelement releases. We can add a comment regarding the existing [issue](https://github.com/mediaelement/mediaelement/issues/2949) in the Less.
